### PR TITLE
Azure build for the project

### PR DESCRIPTION
**Pull request by jeffin143**

### CODE REVIEW

**Comment by zoq** [comment]
Enabled the pipeline, I guess once you commit something we will see if it runs correctly. Maybe you have to merge the latest master branch.

File: .ci/linux-steps.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+steps:
+# Checkout repository
+- checkout: self
+  clean: true
+  fetchDepth: 1
+
+# Install protobuf
+- script: |
+    sudo apt-get install protobuf
```
Comment (zoq) [inline .ci/linux-steps.yaml]:
```suggestion
    sudo apt-get update
    sudo apt-get install -y protobuf
```

**Comment by zoq** [comment]
Maybe `snap install protobuf` works?

File: .ci/linux-steps.yaml (hunk 1 above)
Comment (jeffin143) [reply .ci/linux-steps.yaml]: threw same error not apt package found snap package found

**Comment by jeffin143** [comment]
@zoq  : Finally done

File: include/proto/event.proto [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+syntax = "proto3";
+
+package mlboard;
+
+import "summary.proto";
+
+option cc_enable_arenas = true;
+
+// Protocol buffer representing an event that happened during
+// the execution of a Brain model.
```
Comment (toshal-a) [inline include/proto/event.proto:11]: What is brain model?

**Review by toshal-a — APPROVED** [review]
Hi, this looks good to me. From where have you taken the proto files. I hope they are from the tensorboard's repo.

**Comment by toshal-a** [comment]
I cross checked it by myself. Looks good. 
Thanks @jeffin143  and @zoq .

**Pull request merged**